src/m365/spo/commands/file/file-get.spec.ts

```diff
@@ -501,7 +500,7 @@ describe(commands.FILE_GET, () => {
 
     setTimeout(() => {
       writeStream.emit('close');
-    }, 5);
+    }, 0);
```

Comment:
What's the idea here? Are you sure 0 ms is enough? Did you test it thoroughly?

Reply:
It makes the test run quicker (even though 5 ms is nothing)

Reply:
Im not sure why 5ms was chosen. Maybe changing it to zero might lead to a timing issue sometimes.

Well, let's keep it at 0 and we'll see